File: js/components/winLose.js
function winorlose(status) {
    console.log("you " + status);

    let choice = prompt("do you want to play again? y/n: ");

    if (choice == "n") {
        console.log("========= see ya!🤩🤩 ==========");
        exit()
    } else if (choice == "y") {
        playerLives = 5;
        computerLives = 5;
        player = false;
    }
}

export { winorlose }

File: gameComponents/winLose.py
from gameComponents import gameVars


def winorlose(status):
    print("you " + status)

    choice = input('\033[36m' + 'do you want to play again?😍 y/n: ')

    if choice == "n":
        print('\033[92m' + '========= see ya!🤩🤩  ==========' + '\033[0m\n')
        exit()
    elif choice == "y":
        gameVars.playerLives = 3
        gameVars.computerLives = 3
        gameVars.player = False